articles/qdk-relnotes.md

```diff
@@ -19,6 +19,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## QDK Version 0.xx.xxxx.xxxxxx
+
+*Release date: June 22nd, 2021*
+
+- You can now configure how many solutions they want returned from a solver run (see "Returning multiple solutions" [section](https://docs.microsoft.com/en-us/azure/quantum/optimization-usage-problem-apply-solver-to-optimization))
+- A new NuGet package Microsoft.Quantum.AutoSubstitution, which when added to a Q# project, allows you to annotate operations with the SubstitutableOnTarget(AltOp, Sim) attribute. It will then call AltOp instead of the annotated operation, whenever it is executed using Sim.
+- Integration with Azure-Identity provides more mechanisms to authenticate with Azure.
```

Comment:
@ffrachon found an issue when switching between accounts, we found the following workaround:
```suggestion
- Integration with Azure-Identity provides more mechanisms to authenticate with Azure.
    - If you are using subscriptions on more than one MSA account, in order to avoid issues when switching back and forth between subscriptions, please follow the following one-time manual installation steps:
        1. Upgrade Azure CLI manually to the latest version (currently [2.25.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)). Do not use `az upgrade`.
        2. Run `az login` to login to your account.
        3. Run `az logout` and `az login` to switch to your other account.
        4. You can now run your payloads on the desired subscriptions without having to log out and back in again.
```

Reply:
@guenp is there anywhere else we should document this than the relnotes?

Reply:
I don't think we should. If anything, this is a problem with the az cli, not ours.